MetaMask/metamask-mobile
# fix: validate chain before send

## **Description**

This PR validates the chain id before handling the deeplink for making an ethereum send.

## **Related issues**

Fixes: [11966](https://github.com/MetaMask/metamask-mobile/issues/11966)

## **Manual testing steps**

1. Open the deeplink https://metamask.app.link/send/0x2990079bcdEe240329a520d2444386FC119da21a@56?value=3e18
2. Ensure that BNB is not added as a chain on the wallet
3. You should see an error "Unable to find network with chain id"

## **Screenshots/Recordings**





https://github.com/user-attachments/assets/c790fcdc-7737-4718-b5b5-f117151ea0a2




## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
